apiary.apib

```diff
@@ -2249,7 +2249,7 @@ Get a list of deals.
                 + responsible_user_id: `98b2863e-7b01-4232-82f5-ede1f0b9db22`, `98b2863e-7b01-4232-82f5-ede1f0b9db24` (array[string], optional)
             + updated_since: `2018-02-11T16:45:30+00:00` (string, optional) - This date is inclusive
             + created_before: `2018-02-11T16:45:30+00:00` (string, optional) - This date is inclusive
-            + status: `open`, `won`, `lost` (array[string], optional)
+            + status: `new`, `open`, `won`, `lost` (array[string], optional)
```

Comment:
Code wise it is not allowed, so that will require some changes too

Reply:
Ah, I didn't check the validator. Indeed not allowed at the moment, which doesn't really make sense I guess?

Reply:
Actually, filtering on `open` also returns `new` deals...

Reply:
Oh, then I'm not sure if we need to include the new? 🤔

Reply:
Yeah, i'll drop it on the filter